src/main/cocoa/CocoaCairoView.mm

```diff
@@ -82,6 +101,13 @@ - (CGImageRef)renderCairoImage {
     return image;
 }
 
+- (void)startRedrawLoop {
```

Comment:
Do I right understand that you launch a redraw timer with 60 Hz rate?
If it is true, I think we need to handle the case when window does not need a redraw to no to excessively eat CPU/GPU resources for rendering.